src/index.html

```diff
@@ -11,6 +11,27 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div>
+      <form action="#"
+        class="search-form"
+        method="get"
+        data-qa="big">
+        <input type="text"
+          class="search-input"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress">
+      </form>
```

Comment:
```suggestion
      <form action="#"
        class="search-form"
        method="get"
        data-qa="big"
      >
        <input type="text"
          class="search-input"
          placeholder="Try &quot;Los Angeles&quot;"
          data-qa="keypress"
        >
      </form>
```
close bracket of tag should have similar indent as open. check all similar cases